### DISCUSSION TITLE
Change the language of the creation/modification date

### DISCUSSION THREAD

**Discussion by IwishIcanFLighT** [structurizr/lite#109] [Q&A] [answered]
Hey there!

I'm currently building the architecture diagrams of my project using the C4 Model. Really liking it so far! 👍 

I use Structurirz Lite in Docker, which appends a creation/modification date in French in the bottom left corner of my diagrams. I'd like this date to be in English as the rest of my documentation.
I assume Structurirz gets either my OS or Browser language and uses it... Is there a way to force English?

Thanks!

**Comment by simonbrowndotje** [comment, marked as answer]
You can use a viewset property -> https://structurizr.com/help/properties ... for example:

```
workspace {
    
    model {
        user = person "User"
    }
    
    views {
        systemLandscape "key" {
            include *
        }
        
        properties {
            "structurizr.locale" "en-GB"
        }
    }

}
```

**Reply by IwishIcanFLighT** [reply]
It worked! Thank you very much!
